tensorflow/contrib/tensorrt/kernels/trt_engine_op.cc

```diff
@@ -141,9 +136,7 @@ TRTEngineOp::TRTEngineOp(OpKernelConstruction* context)
   OP_REQUIRES_OK(context, context->GetAttr("max_cached_engines_count",
                                            &max_cached_engines_));
   // Set capacity for LRU cache.
-  lru_cache_.reserve(max_cached_engines_);
-  OP_REQUIRES_OK(context,
-                 context->GetAttr("fixed_input_size", &fixed_input_size_));
+  //lru_cache_.reserve(max_cached_engines_);
```

Comment:
Why removing this?

Reply:
L44: Since LRUCache is now created by LookupOrCreate ` auto status = res_mgr->LookupOrCreate(
      funcdef_name_, "EngineCache", &cache_res,
      {[this](TRTEngineCacheResource** cr) -> tensorflow::Status {
        LOG(INFO) << "max cached engines: " << this->max_cached_engines_;
        *cr = new TRTEngineCacheResource(this->max_cached_engines_);
        return Status::OK();
      }});`

L45-46: fixed_input_size is an unused attribute so I am removing it